<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSettingsMenu</class>
 <widget class="QWidget" name="ControlSettingsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="10" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="ForwardButton">
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="LeftButton">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Move forward</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Move left</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="BackButton">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Move back</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Move right</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="RightButton">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Jump</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="BackButton_4">
       <property name="text">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="BackButton_5">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Use</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="BackButton_6">
       <property name="text">
        <string>RShift</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Dig/Attack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BackButton_7">
       <property name="text">
        <string>LMB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Set Block</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BackButton_8">
       <property name="text">
        <string>RMB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Control Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="BackToSettings">
     <property name="text">
      <string>Back to Settings</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Set Everything Default</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QPushButton" name="BackToMainMenu">
     <property name="text">
      <string>Back to main menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
